app/templates/statistiky.html

```diff
@@ -96,6 +96,7 @@ <h5>Nedovolené zjednodušení</h5>
                         title="Dataset registrovaných dle profesí.">Stav registrací dle prioritních skupin</h5>
                     {{ stav_registraci_kategorie(reservations_category) }}
                     <p class="card-text font-italic mt-3">
+                        28. 4. z historických dat jsme rekonstruovali staré údaje. (data ze systému mizí po očkování) <br />
```

Comment:
I would remove both notes and leave it only in news

Reply:
Good idea

Reply:
The same note is probably also below some graph, remove it too please